drivers/sensor/mcp9808/mcp9808.c

```diff
@@ -97,9 +136,10 @@ static const struct mcp9808_config mcp9808_cfg = {
 	.alert_pin = DT_INST_GPIO_PIN(0, int_gpios),
 	.alert_flags = DT_INST_GPIO_FLAGS(0, int_gpios),
 	.alert_controller = DT_INST_GPIO_LABEL(0, int_gpios),
+	.resolution = DT_INST_PROP(0, resolution),
 #endif /* CONFIG_MCP9808_TRIGGER */
 };
 
 DEVICE_DT_INST_DEFINE(0, mcp9808_init, device_pm_control_nop,
-		    &mcp9808_data, &mcp9808_cfg, POST_KERNEL,
-		    CONFIG_SENSOR_INIT_PRIORITY, &mcp9808_api_funcs);
+		      &mcp9808_data, &mcp9808_cfg, POST_KERNEL,
+		      CONFIG_SENSOR_INIT_PRIORITY, &mcp9808_api_funcs);
```

Comment:
is this change needed to align?

Reply:
This is the output from the default uncrustify. I presume this change is correct.